macros/WebDriverSidebar.ejs

```diff
@@ -0,0 +1,110 @@
+<%
```

Comment:
The `<% … %>` section contains JavaScript code to execute, everything outside it is HTML.

You did this in the exact opposite way to how EJS (Embedded JavaScript templates) work.

Reply:
Also, you can run the tests without docker by running `node run.js`, and then, while the server is running, run `./node_modules/.bin/jshint --show-non-errors lib tests` and `./node_modules/.bin/mocha tests` in a separate console window.